mlrun/common/schemas/alert.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from datetime import datetime
+from typing import Annotated, Optional, Union
+
+import pydantic
+
+from mlrun.common.schemas.notification import Notification
+from mlrun.common.types import StrEnum
+
+
+class EventEntityKind(StrEnum):
+    MODEL = "model"
+    JOB = "job"
+
+
+class EventEntity(pydantic.BaseModel):
+    kind: EventEntityKind
+    project: str
+    id: str
+
+
+class EventKind(StrEnum):
+    DRIFT_DETECTED = "drift_detected"
+    DRIFT_SUSPECTED = "drift_suspected"
```

Comment:
Ok but can you add another kind like I mention?
@theSaarco what do you think?

Reply:
We have discussed adding something generic like `METRIC_THRESHOLD_PASSED` that can be used for custom metrics to alert on passing a threshold. It can be added here or in subsequent PR, depending on when you need this type of event in the code.

Reply:
Sounds good, lets discuss it before and agree on the name and the relevant values :)